public/configs/cctp.json

```diff
@@ -1,46 +1,57 @@
 [
     {
         "chainId": "42161",
+        "token": "0xaf88d065e77c8cC2239327C5EDb3A432268e5831",
```

Comment:
can we rename to `addressOrDenom`

Reply:
It should just be the address.  For Noble chain, Squid returns `"address": "uusdc"`, so I am just copying it.

Reply:
address works!